app.js

```diff
@@ -12,8 +13,11 @@ api.socket.initSocket({ socket: config.socket, wsType: config.ws_type, onNewMess
 setTimeout(init, 5000);
 
 function init() {
-  require('./server');
   try {
+    if (config.api?.port) {
```

Comment:
```suggestion
    if (config.api.port) {
```
api.port: `isRequired = true`